mongoexport/main/mongoexport.go

```diff
@@ -40,19 +40,20 @@ func main() {
 	if err != nil {
 		log.Logvf(log.Always, "%v", err)
 
-		if se, ok := err.(mongoexport.SetupError); ok {
-			os.Exit(se.Code)
+		if se, ok := err.(util.SetupError); ok {
+			if se.Message != "" {
+				log.Logv(log.Always, se.Message)
```

Comment:
How is this different than err.String() and should we combine reporting the error (line 41) with this rather than a separate log line?  I.e. either "error: ..." or "error: ... : <setup message>"?